molecule.yml

```diff
@@ -0,0 +1,19 @@
+---
+ansible:
```

Comment:
You're right.`check` does a dry run,  so it's not running the `raw` command that populates the variable in the `debug` message. Addressed in [23e85cc](https://github.com/azavea/ansible-ntp/pull/2/commits/23e85cc080b048a3078a0fb0222c4337497f5d1a).